File: D2 Problems/1986_지그재그숫자.py
T = int(input())
for test_case in range(1, T + 1):
    n = int(input())
    sum = 0
    for x in range(1, n + 1):
        if x % 2 == 0:
            sum -= x
        else:
            sum += x
    print("#%d %d" %(test_case, sum))

File: 1986_지그재그숫자.cpp
#include<iostream>

using namespace std;

int main(int argc, char** argv)
{
	int test_case;
	int T;
	int n, i;
	int sum;
	cin >> T;
	for (test_case = 1; test_case <= T; ++test_case)
	{
		cin >> n;
		sum = 0;
		for (i = 1; i <= n; ++i)
		{
			if (i % 2 == 0)
				sum -= i;
			else
				sum += i;
		}
		cout << "#" << test_case << " " << sum << "\n";
	}
	return 0;
}